manifests/init.pp

```diff
@@ -13,8 +13,9 @@
   $negatives = getvar('facts.pe_status_check', []).filter | $k, $v | { $v == false and ! ($k in $indicator_exclusions) }
 
   $negatives.each |$indicator, $_v| {
+    $in_message = lookup("pe_status_check::${indicator}")
```

Comment:
Users that still have hiera 3 enabled will see failures here unless there is a default specified. Please add a default value in the lookup.

Reply:
OK that should be done